### PR TITLE
cicd: add script to edit module for nightly

### PR DESCRIPTION
**Pull request by hdonnay**
This script should automate editing the clair module so that it builds
against claircore's main branch. The idea is to run this in a CI job,
build the resulting code, and then throw away the modifications.

Signed-off-by: Hank Donnay <hdonnay@redhat.com>

### CODE REVIEW

**Comment by hdonnay** [comment]
You can see some successful runs [here](https://github.com/hdonnay/clair/actions/workflows/nightly.yml).

File: Dockerfile [hunk 1]
```diff
@@ -24,7 +25,12 @@ RUN go build\
 
 FROM registry.access.redhat.com/ubi8/ubi-minimal AS final
 RUN microdnf install --disablerepo=* --enablerepo=ubi-8-baseos --enablerepo=ubi-8-appstream tar
-RUN curl -L -o /usr/local/bin/dumb-init https://github.com/Yelp/dumb-init/releases/download/v1.2.2/dumb-init_1.2.2_amd64 && chmod +x /usr/local/bin/dumb-init 
+RUN case "$(uname -m)" in \
```
Comment (crozzy) [inline Dockerfile:28]: this is a little bit of a shame, but can't think of a more graceful way

**Review by crozzy — APPROVED** [review]
awesome, updating my cluster images...

File: Dockerfile (hunk 1 above)
Comment (hdonnay) [reply Dockerfile:28]: Yeah, I don't know when the syntax/backend updates that allow the "buildx" extra variables will be usable.

Comment (crozzy) [reply Dockerfile:28]: ...or rhel pick up / yelp publish rpms and microdnf would take care of the arch parsing

Comment (hdonnay) [reply Dockerfile:28]: Yeah, I don't recall if there's an intent to package `dumb-init,` and I certainly don't want to be on the hook for an `rpm`.

The other options are:

1. write a little bit of code to catch `SIGCHLD` and ditch `dumb-init`
2. remove the exec calls that necessitate pid 1 handling (currently just `rpm`, I think)

**Pull request merged**